crates/bevy_window/Cargo.toml

```diff
@@ -1,21 +1,22 @@
 [package]
-name = "bevy_window"
-version = "0.8.0"
-edition = "2021"
 description = "Provides windowing functionality for Bevy Engine"
+edition = "2021"
 homepage = "https://bevyengine.org"
-repository = "https://github.com/bevyengine/bevy"
-license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
+license = "MIT OR Apache-2.0"
+name = "bevy_window"
+repository = "https://github.com/bevyengine/bevy"
+version = "0.8.0"
 
 [dependencies]
 # bevy
-bevy_app = { path = "../bevy_app", version = "0.8.0" }
-bevy_ecs = { path = "../bevy_ecs", version = "0.8.0" }
-bevy_math = { path = "../bevy_math", version = "0.8.0" }
-bevy_utils = { path = "../bevy_utils", version = "0.8.0" }
+bevy_app = {path = "../bevy_app", version = "0.8.0"}
+bevy_ecs = {path = "../bevy_ecs", version = "0.8.0"}
+bevy_math = {path = "../bevy_math", version = "0.8.0"}
+bevy_reflect = {path = "../bevy_reflect", version = "0.8.0"}
+bevy_utils = {path = "../bevy_utils", version = "0.8.0"}
 # Used for close_on_esc
-bevy_input = { path = "../bevy_input", version = "0.8.0" }
+bevy_input = {path = "../bevy_input", version = "0.8.0"}
```

Comment:
Not for this PR, but I wonder if those utility-systems such as "close on escape" should be extracted out to a separate crate?

IMO it has more to do with general app behaviour than windowing

Reply:
Ya I was thinking about it when going through there but wasn't entirely sure where I would put them.